kubernetes/linux/logrotate.conf

```diff
@@ -37,3 +37,23 @@
     compress
     size 10M
 }
+
+/var/opt/microsoft/docker-cimprov/log/fluent-bit.log {
+    copytruncate
+    rotate 7
+    missingok
+    notifempty
+    delaycompress
+    compress
+    size 10M
+}
+
+/var/opt/microsoft/docker-cimprov/log/fluent-bit-geneva.log {
+    copytruncate
+    rotate 7
+    missingok
+    notifempty
+    delaycompress
+    compress
+    size 10M
+}
```

Comment:
add new line at the end? and also, can you please deploy and verify log rotation indeed working?